### ISSUE TITLE
Fatal Build Error

### ISSUE THREAD

**Issue by werbeckes** [bugsnag/bugsnag-react-native#176]
I'm getting the following error when building our project:
```
../node_modules/bugsnag-react-native/cocoa/vendor/bugsnag-cocoa/Source/BugsnagNotifier.m:50:42: error: initializer element is not a compile-time constant
NSString *const BSAttributeBreadcrumbs = BSGKeyBreadcrumbs;
                                         ^~~~~~~~~~~~~~~~~
```

Saw this error when we were on version 2.4.2.  Upgraded to 2.5.3 and am still getting this error and unable to build.  Thanks for your help.

ps. This might be unrelated, but I'm also getting a lot these 'notes':
```
../node_modules/bugsnag-react-native/cocoa/vendor/bugsnag-cocoa/Source/BugsnagCrashReport.h:156:51: note: setter and getter must both be synthesized, or both be user defined,or the property must be nonatomic
@property(readwrite, copy, nonnull) NSDictionary *metaData;
                                                  ^
```

**Comment by fractalwrench** [comment]
Thanks for the report @werbeckes - a temporary workaround for this should be replacing `BSGKeyBreadcrumbs` with `@"breadcrumbs"`.

Do you have any custom build settings for your project target? You should be able to check this by opening your iOS xcodeproj and searching for "warnings" in Build Settings > Target.

I'm having trouble reproducing the error on our example projects, and it would be good to confirm that this is the only issue before making a new release. Are you installing via npm?

**Comment by opula** [comment]
I had the same issue.  The workaround worked for me!

**Comment by fractalwrench** [comment]
Resolved in v2.5.4 - it appears that XCode 9 doesn't detect this as a build error.
